Utils/utils.R

```diff
@@ -40,31 +40,69 @@ create_db <- function(){
   
   # Apply each query.
   sapply(queries, function(x){
-    res <- dbSendStatement(
-      con,
-      paste(scan(x, sep = "\n", what = "character"), collapse = ""))
     
-    dbClearResult(res)
+    tryCatch({
+      rs <- dbSendQuery(con, query)
+    }, error = function(err) {
+      message <- paste("dbSendQuery returned",err)
+      message(message, .loggit = FALSE)
+      loggit("ERROR", message)
+      dbDisconnect(con)
+    })
+    
+    dbClearResult(rs)
```

Comment:
Hi @Robert-Krajcik, when I run the app after removing the `database.sqlite` and the `loggit.json` files, I get the following error:

`dbSendQuery returned Error in dbSendQuery(con, query): object 'query' not found`

followed by 

` Error in dbClearResult(rs) : object 'rs' not found `

Reply:
OK I believe I have this figured out, @marlycormar